Makefile

```diff
@@ -5,6 +5,7 @@ TRIVY_SCANNER_IMG ?= ghcr.io/azure/eraser-trivy-scanner:${VERSION}
 MANAGER_IMG ?= ghcr.io/azure/eraser-manager:${VERSION}
 ERASER_IMG ?= ghcr.io/azure/eraser:${VERSION}
 COLLECTOR_IMG ?= ghcr.io/azure/collector:${VERSION}
+VULNERABLE_IMG ?= docker.io/library/nginx:1.21.6
```

Comment:
this one doesn't have any fixed vulns (--ignore-fixed-vulns flag), should we use something like `docker.io/library/alpine:3.7`?